lambdas/submit-answer/src/services/save-results-service.ts

```diff
@@ -0,0 +1,60 @@
+import { Logger } from "@aws-lambda-powertools/logger";
+import { SubmitAnswerResult } from "../types/answer-result-types";
+import { AnswerResultItem } from "../types/answer-result-types";
+import {
+  DynamoDBDocument,
+  GetCommand,
+  PutCommand,
+} from "@aws-sdk/lib-dynamodb";
+
+const ServiceName: string = "SaveQuestionsService";
```

Comment:
Service name does not match class name.
As this is doing save and get can we just rename it results-service